allennlp/data/dataset_readers/stanford_sentiment_tree_bank.py

```diff
@@ -0,0 +1,136 @@
+from typing import Dict, List
+import logging
+
+from overrides import overrides
+from nltk.tree import Tree
+import tqdm
+
+from allennlp.common import Params
+from allennlp.common.file_utils import cached_path
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.fields import LabelField, TextField, Field
+from allennlp.data.instance import Instance
+from allennlp.data.token_indexers import TokenIndexer, SingleIdTokenIndexer
+from allennlp.data.tokenizers import Token
+from allennlp.common.checks import ConfigurationError
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+@DatasetReader.register("sst_tokens")
+class StanfordSentimentTreeBankTokensDatasetReader(DatasetReader):
```

Comment:
Why have `Tokens` in the name here?  Doesn't seem like it belongs.  Oh, is it because you could read this as trees, and have some tree encoder?  I'd still take out `Tokens` from the name, and make the case where you're returning trees as the unusual one with a longer name (or just add it as an optional, additional output here, if we ever want that).  When you read the PTB, you get tokens as input and trees as output, so it's not unexpected to get tokens when you see a "treebank" reader.